aten/src/ATen/native/native_functions.yaml

```diff
@@ -3960,7 +3960,7 @@
     CPU: narrow_copy_dense_cpu
     SparseCPU, SparseCUDA: narrow_copy_sparse
     CompositeExplicitAutogradNonFunctional: narrow_copy_dense_symint
-  tags: view_copy
+  tags: [canonical, view_copy]
```

Comment:
Sorry, I recently renamed this to Core Aten Ops

https://github.com/pytorch/pytorch/pull/92904